governance/reviews/sandbox.md

```diff
@@ -0,0 +1,92 @@
+# Sandbox Project Review Guide
+
+Every 1-2 months, the TOC reviews a group of projects which have applied for Sandbox Project membership in the CNCF.  It is the duty of TAG Contributor Strategy to help the TOC with a quick check on how well set up these projects are to be projects, in order to help the TOC make a decision on them.
+
+Note that there are very few requirements for Sandbox projects for governance or contributor information.  As such, our evaluation is merely advisory, and not qualifying.  It should also take less than 20 minutes per project.
+
+You can find the Sandbox projects to be reviewed next on [this project board](https://github.com/orgs/cncf/projects/14/views/1), in the column “Upcoming”.
+
+## Template
+
+For each project review, create a comment based on the following template:
+
+```
+TAG Contributor strategy has reviewed this project and found the following:
+
+* The [contributor guide](link) is {basic|average|exceptionally detailed}.
+* The [governance](link) is {description of general governance} and is {minimal|complete|exceptionally detailed}
+* The [roadmap](link) is {describe type of roadmap} and appears {new|very active|minimal|unused}
+* There are {number} [maintainers](link), who work for {describe employer distribution}
+* {Other comments about project governance or apparent contributor activity}
+
+This review is for the TOC’s information only.  Sandbox projects are not required to have full governance or contributor documentation.
+```
+
+[Example review](https://github.com/cncf/sandbox/issues/87#issuecomment-2264280131), based on this template.
+
+## How to Fill Out The Template
+
+As you can see, we’re checking minimal things because we have limited expectations of Sandbox projects.  The four things we’re checking are:
+
+### Contributor Guide
+
+Does the project have a contributor guide?  If so, how complete/useful does it appear to be?
+
+Here you’re checking for a contributor guide, and whether it covers all of the aspects of contribution or just one or two of them.  For an example of a relatively complete Contributor Guide, see our template.  Note that for very complete contribution guides, there may be multiple files, including a release guide, testing guide, reviewing guide, etc.  Make sure you check for links to other documentation.  Contributor documentation may also be part of the project’s user documentation instead of being in a git repo.
```

Comment:
```suggestion
Here you’re checking for a contributor guide, and whether it covers all of the aspects of contribution or just one or two of them.  For an example of a relatively complete Contributor Guide, see [our template](https://contribute.cncf.io/maintainers/templates/contributing/).  Note that for very complete contribution guides, there may be multiple files, including a release guide, testing guide, reviewing guide, etc.  Make sure you check for links to other documentation.  Contributor documentation may also be part of the project’s user documentation instead of being in a git repo.
```